### ISSUE TITLE
md-nav-item should be able to input aria-label

### ISSUE THREAD

**Issue by alanpurple** [angular/material#10110]
md-nav-item needs aria label so we can use md-icon inside the md-nav-item block

for now, there's no way to put aria-label in the anchor block generated from md-nav-item
